ioctl/newcmd/action/action_test.go

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotexapi/mock_iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+)
+
+const (
+	veryLightScryptN = 2
+	veryLightScryptP = 1
+)
+
+var (
+	testData = []struct {
+		endpoint string
+		insecure bool
+	}{
+		{
+			endpoint: "111:222:333:444:5678",
+			insecure: false,
+		},
+		{
+			endpoint: "",
+			insecure: true,
+		},
+	}
+)
+
```

Comment:
Is `TestSendAction`  removed ?

Reply:
If codes is too large. we can split to small prs: one or two TestXxx  per pr. What do you think of it ?

Reply:
`TestSendAction` has removed temporily, since it isn't completed yet.
I think sperate PRs will be better. `TestSendRaw` and `TestSigner` are alike, they can be in same PR. `TestSendAction`, `TestExecute` and `TestRead` and similar, they can be in a PR.

Reply:
I'll open another PR for the rest 3 test case.

Reply:
@LuckyPigeon  Ok.